src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,41 @@
      * Андрей - 600
      * София - 900</p>
      */
+    final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        StringBuilder str = new StringBuilder("");
```

Comment:
```suggestion
        StringBuilder str = new StringBuilder();
```
you can instantly put a part of message into constructor or leave it empty but there is no need to write an empty string as it may be created by-default